packages/react-pose/es/base.js

```diff
@@ -0,0 +1,374 @@
+import { __rest, __assign, __extends } from 'tslib';
```

Comment:
personally I would `.gitignore` this file, but similar directories (`dist`) in other packages in this repo are not ignored, so I left it for now.